VERSION

```diff
@@ -1 +1 @@
-2.4.0
+2.4.1
```

Comment:
We are upgrading across major versions of terraform which would require some migration work to move existing resources from terraform v13 -> 14 -> 15, and would not be backwards compatible.

I think this warrants a major version increase instead of just a patch version increase, but I will leave the final choice to you.

https://www.terraform.io/language/upgrade-guides/0-15#upgrading-to-terraform-v0-15